pkg/k8s/node.go

```diff
@@ -65,6 +67,10 @@ func ParseNode(k8sNode *slim_corev1.Node, source source.Source) *nodeTypes.Node
 			continue
 		}
 
+		if addr.Type != apiv1.NodeExternalIP {
+			scopedLog.WithError(err).Warn("preffered address type not found")
```

Comment:
```suggestion
			scopedLog.WithError(err).Warn("preferred address type not found")
```

Reply:
@tommyp1ckles Could you please inform me whether the change is reasonable or not?
If not, could you please give some idea about the correct approach.

Reply:
Sorry, was just familiarizing myself with the issue at hand. Thanks for the PR.

I believe the issue is that k8s Nodes are expected to have at most one IP of each type (i.e. 1 ExternalIP, 1 InternalIP, ...).

I think your approach will warn whenever any IP not of type "ExternalIP" is found, I don't think we necessarily want to warn in this situation, unless another instance of the same address type has already been found.

Perhaps we should keep track of what types have already been found, and then warn if we see a duplicate?